src/types/auth.ts

```diff
@@ -0,0 +1,37 @@
+/** Authenticate with Mercury as a User */
```

Comment:
based on our naming, this file would be `src/types/auth.types.ts` (don't mind I'm commenting on the first line, not sure how to comment on file name)